mlrun/projects/project.py

```diff
@@ -173,9 +320,20 @@ def _load_project_dir(context, name="", subpath=""):
     return project
 
 
-def _load_project_from_db(url, secrets):
+def _add_username_to_project_name_if_needed(name, user_project):
```

Comment:
I would have the `set_environment` function call this function - this way we have a single place to modify once we want to do project name normalization.

Reply:
done